src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    /**
+     * Constructs an instance of Deadline
+     *
+     * @param description Description String
+     * @param by Date String
+     */
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = DateTimeConverter.formatDate(by);
+    }
+
+    /** @inheritdoc */
+    @Override
+    public String changeFormat() {
+        String done = isDone ? "1" : "0";
```

Comment:
Any reason of doing this rather than changing isDone and use the usual toString function instead?